lib/cyclone-client/src/client.rs

```diff
@@ -1325,6 +1325,7 @@ mod tests {
         let req = ManagementRequest {
             execution_id: "1234".to_string(),
             handler: "manage".to_string(),
+            current_view: "DEFAULT".to_string(),
```

Comment:
The "default view" is named DEFAULT, but that is editable. It might be better to find the default view and send its name??

Reply:
This is just in the test harness. We need something there (anything will do)